### PR TITLE
Make example list of enabling policies

### LINKED ISSUE #793: List of open source policies from governments (moved from publiccode.net repo)
Relevant encouraging policy frameworks:
- About repo issue [716](https://github.com/publiccodenet/about/issues/716)
- About repo issue [630](https://github.com/publiccodenet/about/issues/630) and 
- About repo issue [572](https://github.com/publiccodenet/about/issues/572)) 

Users:
- general public
- awareness raising with the general public
- fan at public organizations (#71)

From: communications assets for membership workshop, 23 July 2020

### PR DESCRIPTION
**Pull request by Ainali**
Fixes #793

### CODE REVIEW

File: activities/understanding-our-users/enabling-policies.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+type: Resource
+---
+
+# Enabling policies
+
+If there are policies supporting open source development it can be easier for civil servants to convince their peers that they should engage.
+These policies can take many forms, from laws to internal policies.
+The clearer and more specific the policy is, the likelier it is that it will be a helpful argument.
+
+Some exampels (non-exhaustive list):
```
Comment (ericherman) [inline activities/understanding-our-users/enabling-policies.md:13]: examples

File: activities/understanding-our-users/index.md [hunk 2]
```diff
@@ -12,10 +12,14 @@ redirect_from:
...
 This set of resources helps to understand our users, and how our work does or may impact them.
 
-## Existing pain Points
+## Existing pain points
 
 * Understanding the [pain points that public organizations](public-organizations.md) have whith their digital infrastructure
 
+## Existing enabling policies
+
+* Understanding the types of [enabling policies](enabling-policies.md) that can help public organizations and civil servants
```
Comment (ericherman) [inline activities/understanding-our-users/index.md:21]: Missing trailing period.

**Review by ericherman — CHANGES_REQUESTED** [review]

File: activities/understanding-our-users/index.md (hunk 2 above)
Comment (Ainali) [reply activities/understanding-our-users/index.md:21]: Yes, I was unsure what to do on this page. All other bullet points, except the last one under Additional resources, also miss it. Should it be added to all, only here, or removed on the single instance where it is?

Comment (ericherman) [reply activities/understanding-our-users/index.md:21]: I agree, as it is a bullet point, it should be consistent with the others.

**Review by ericherman — DISMISSED** [review]

File: activities/understanding-our-users/enabling-policies.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+---
+# SPDX-License-Identifier: CC0-1.0
+# SPDX-FileCopyrightText: 2023 The Foundation for Public Code <info@publiccode.net>
+type: Resource
+---
+
+# Enabling policies
+
+If there are policies supporting open source development it can be easier for civil servants to convince their peers that they should engage.
```
Comment (ElenaFdR) [inline activities/understanding-our-users/enabling-policies.md:9]:
```suggestion
Policies supporting open source development can make it easier for civil servants to convince their peers that they should engage.
```

File: activities/understanding-our-users/enabling-policies.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+These policies can take many forms, from laws to internal policies.
```
Comment (ElenaFdR) [inline activities/understanding-our-users/enabling-policies.md:10]: This lets us avoid a 'policies can take many forms, including policies' recursive argument.
```suggestion
These policies can take many forms, from laws to internal guidance.
```

**Review by ElenaFdR — CHANGES_REQUESTED** [review]
I've made some linguistic edits, plus made a suggestion re more examples.

**Review by ericherman — APPROVED** [review]

File: activities/understanding-our-users/enabling-policies.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+# SPDX-License-Identifier: CC0-1.0
+# SPDX-FileCopyrightText: 2023 The Foundation for Public Code <info@publiccode.net>
+type: Resource
+---
+
+# Enabling policies
+
+Policies supporting open source development can make it easier for civil servants to convince their peers that they should engage.
+These policies can take many forms, from laws to internal guidance.
+The clearer and more specific the policy is, the more likely that it is that it will persuade skeptics.
```
Comment (ElenaFdR) [inline activities/understanding-our-users/enabling-policies.md:11]:
```suggestion
Clearer and more specific policies will be more likely to persuade skeptics.
```

**Review by ElenaFdR — APPROVED** [review]
Much better! I've made a vastly simplifying suggestion on the only sentence that's still a bit clunky.

**Pull request merged**